apps/passport-client/package.json

```diff
@@ -24,6 +24,7 @@
     "@types/pako": "^2.0.0",
     "dotenv": "^16.0.3",
     "json-bigint": "^1.0.0",
+    "jubjub-signature-pcd": "^0.0.1",
```

Comment:
```suggestion
    "jubjub-signature-pcd": "0.0.1",
```

For security, I'd prefer to lock this to a particular version, then updated it explicitly when there are updates.

Reply:
Locked to version `^0.0.2` [here](https://github.com/proofcarryingdata/zupass/pull/154/commits/599131a7a6683025623a4a50f3ab0b37e0fe557b)